tox.ini

```diff
@@ -0,0 +1,92 @@
+[tox]
+envlist = py{39,310,311,312,313,314,py310}
+labels =
+    check = typing, gen_exports, type_completeness, pip_compile
+    cython = py39-cython2,py39-cython,py311-cython2,py313-cython
+
+# TODO:
+# * environment to check coverage
+# * replace ci.sh
+#   * --verbose --durations=10
+#   * -p _trio_check_attrs_aliases
+#   * mypy cache
+#   * LSP
+#   * apport
+# * use tox in CI
+# * switch to nox?
+# * move to pyproject.toml?
+#   * this means conditional deps need to be replaced
+
+# protip: install uv-tox for faster venv generation
+
+[testenv]
```

Comment:
Could you add `description =` to every env? It should explain what they do and would be printed out via `tox l` or `tox -av`.